apps/catalogue/src/components/detailView/OntologyFieldValue.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <ObjectDisplay
+    :class="'font-weight-bold mr-2 mb-2 badge bade-lg badge-' + color"
+    :data="data"
+    :metaData="metaData"
+  />
+</template>
+
+<script>
+import { ObjectDisplay } from "@mswertz/emx2-styleguide";
+export default {
+  name: "OntologyFieldValue",
+  components: { ObjectDisplay },
+  props: {
```

Comment:
I would probably call this prop 'value', why did you chose 'data'?

Reply:
because its paired with 'data' , meta -> data